src/VideoRemise/LightDisplayVisualEffect/LightDisplayVisualEffect.csproj

```diff
@@ -27,6 +27,7 @@
     <DefineConstants>DEBUG;TRACE;NETFX_CORE;WINDOWS_UWP</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Where are the unsafe blocks being used?

Reply:
In the visual effect, I think there might be an unsafe block. Or at least an unsafe line or two...